lxd/instance/drivers/driver_qemu.go

```diff
@@ -4907,7 +4909,7 @@ func (d *qemu) cpuTopology(limit string) (int, int, int, map[uint64]uint64, map[
 		nrCores = countCores
 		nrThreads = countThreads
 	} else {
-		logger.Warnf("Instance '%s' uses a CPU pinning profile which doesn't match hardware layout", project.Instance(d.Project(), d.Name()))
+		d.common.logger.Warn(fmt.Sprintf("Instance '%s' uses a CPU pinning profile which doesn't match hardware layout", project.Instance(d.Project(), d.Name())))
```

Comment:
@kevtheappdev when you see historical use of embedding context variables into the log messages themselves, like here, can you replace with proper context logging. Which actually means in this case there is no need to add the project/instance variable as they are already set in the driver's logger.